test/which.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+require('assert');
+const should = require('should');
+const cash = require('../dist/index.js');
+
+describe('true', function () {
+  it('should exist and be a function', function () {
+    should.exist(cash.which);
+  });
+
+  it('should do nothing when called', function () {
+    const results = cash.which();
+    console.log(typeof results);
+    results.should.equal('');
+  });
+
+  it('should find cmd when called', function () {
+    const results = cash.which('cmd');
+    console.log(typeof results);
+    results.should.not.equal('');
+  });
```

Comment:
It should be pretty easy tests for basic functionality. Not trying to re-handle all of the tests that the lib already provides, but a basic test or two that verifies the author didn't butcher the package on the latest update would be good.